fmi-cap-profile.md

```diff
@@ -11,22 +11,22 @@ Specification URIs:
 
 ## Overview
 
-This document defines CAP profile for warnings issued by [Finnish Meteorological Institute](https://www.ilmatieteenlaitos.fi/). The profile extends the [MetCoOp CAP profile v1.0](https://git.smhi.se/metcoop/metcoop-cap-profile/blob/version-1.0/metcoop-cap-profile.md) placing further restrictions while being 100% compatible.
+This document defines a CAP profile for warnings issued by the [Finnish Meteorological Institute](https://www.ilmatieteenlaitos.fi/). The profile extends [MetCoOp CAP profile v1.0](https://git.smhi.se/metcoop/metcoop-cap-profile/blob/version-1.0/metcoop-cap-profile.md) by placing further restrictions while being 100% compatible.
 
 
 ## Profile versioning
 
-This profile is versioned using semantic versioning. Changes in patch and minor versions are backwards-compatible when compatibility notes are followed in client implementation.
+This profile is versioned using semantic versioning. Changes in patch and minor versions are backwards compatible when the client implementation conforms to the compatibility rules.
 
 
-## General compatibility notes
```

Comment:
"Compatibility note" doesn't seem to convey the importance of these rules. But "rules" may not be the best word either..

Reply:
Guideline?
I guess the same word should be used consistently throughout the document.

Reply:
Guideline could be good. But it's a less strong word than "rule" and implies leeway. The way the bullet points are worded make them sound more like rule. I'm thinking "rule" is still better.

Reply:
Ok, let's go with _rule_ then.